cli/pom.xml

```diff
@@ -33,6 +33,11 @@
         <groupId>org.kohsuke</groupId>
         <artifactId>access-modifier-annotation</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci</groupId>
+      <artifactId>annotation-indexer</artifactId>
+      <version>1.12</version>
```

Comment:
Please either use `dependencyManagement`, or remove this dep from `jenkins-core` and have it inherit from `cli`.

Reply:
The explicit dependency would be unnecessary if kohsuke/access-modifier#15 was merged and released and we picked up that version of access-modifier, but `dependencyManagement` seems like a better approach anyway, so +1.